posts/2024-01-05-announcing-stride-4-2-in-dotnet-8.md

```diff
@@ -21,6 +21,49 @@ A massive thank you to the open-source Stride community for your dedicated contr
+
+### Stride's Rising Popularity
+
+It's been an exciting time for us. We've noticed more and more developers giving Stride a try, a trend clearly reflected in our growing user numbers. During the carzy unbity days, we were number 3 in the pupularity rankings, just after Godot and Bevy. Also a big shoutout to gamefromscratch for his video "[Stride For Unity Developers](https://www.youtube.com/watch?v=u_ksFlHHXYU)", which definitely helped.
+
+### The Heartbeat of Stride: Our Community
+
+What really makes Stride tick is you – the community. Your feedback, contributions, and ideas have been the driving force behind Stride 4.2. It’s your hands-on experience and insights that have helped shape this release, making Stride not just a tool but a community achievement.
+
+## Addressing the Pace of Updates in Stride
+
+Let's talk about how often Stride gets updated. We know it's something on your mind, and there's a story behind it.
+
+### The People Behind Stride
+
+Stride is brought to life by a group of passionate developers, much like yourself, who contribute in their free time. This means our pace might not match that of other engines backed by full-time teams, but it also means every update is a labor of love.
+
+### Facing the Realities: Funding and Resources
+
+Here’s the deal: our funding levels shape how much we can do. We’re grateful for every bit of support we receive, but we’re still on our way to having a full-time development force. This financial reality does affect how fast we can roll out updates.
+
+### Making Every Update Count
+
+Although we might not be the fastest out there, we focus on making each update meaningful. Quality, stability, and usability are what we aim for every time we release a new version of Stride.
+
+### Join Us on This Journey
+
+We’re always excited to welcome new contributors to the Stride family. Whether it’s through code contributions, spreading the word, or donations, every bit helps us grow stronger. If you’ve got skills in .NET, Android, and iOS development, there’s a special place for you here. [Support us on OpenCollective](https://opencollective.com/stride3d).
+
+### The Impact of Your Support
+
+We believe that every donation has a substantial impact, perhaps more so than in some other projects. Why? Because C# as a language offers high productivity, and the Stride codebase has a great architecture. This means that every dime you contribute goes a long way.
+
+Consider this: if Stride had the same level of financial support as engines like Godot, we believe our progress would skyrocket. With C#'s productivity and Stride's well-structured code, your donations have the potential to accelerate our development pace significantly. It's not just about keeping the lights on; it's about pushing Stride to new heights, faster and more effectively than many other projects.
```

Comment:
Imagine how much we could do if Stride got as much financial support as other big engines. With C#'s easy productivity and Stride's well-organized code, your donations could help us improve way faster. It's not just about keeping the lights on; it's about making Stride better and faster than many other projects. Your support can take Stride to new heights!

to kick the name godot out of it, as this wasnt taken well last time

Reply:
I agree.

Reply:
Yes, good point!